spec/process_booking_spec.rb

```diff
@@ -76,7 +76,7 @@
     cancel_url = email_link_href(confirmation_email, 'you can cancel this visit')
     visit cancel_url
     expect(page).to have_content('Your visit has been confirmed')
-    check 'Yes, I want to cancel this visit'
```

Comment:
I don't think there is JS interaction on checking the cancel label / checkbox, what broke?